conans/test/functional/command/info_test.py

```diff
@@ -311,6 +313,7 @@ def reuse_test(self):
                 Remote: None
                 URL: myurl
                 License: MIT
+                Description: blah
```

Comment:
How does the output look like for a multi-line description field?

Reply:
I didn't check, lemme add a test for it

Reply:
@memsharded it works. I just updated a test, take a look